lib/src/model.ts

```diff
@@ -63,7 +64,16 @@ export class Model extends EventTarget {
         this._ready = message.result;
       }
 
-      this.dispatchEvent(new CustomEvent(message.event, { detail: message }));
+      const event = new CustomEvent(message.event, { detail: message, ...message });
```

Comment:
Is the extra spreading correct here? If I understand correctly only the [`detail`](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent) property is supported here.

Reply:
You're totally right, this was just a test.